NewHaven/src/Resources.h

```diff
@@ -1,47 +1,69 @@
 #pragma once
+#include <cstdint>
+
+enum struct resourceTypes:std::uint_least8_t { SHEEP, STONE, WHEAT, WOOD };
```

Comment:
this isn't really a complaint or anything, but since you are using the standard namespace a lot why dont you just add using namespace std; after your includes? It would make it so that you don't need to write std:: before calling your function. Its up to you what you prefer. I'll continue to look through the code and double check with the assignments requirements. I should approve it somewhere around friday evening.

Reply:
Also resourceType is defined as a struct and as such the name of the data structure should start with a capital.

Reply:
I prefer not using any namespaces for now just to get familiarized with the content of the standard library. I made the change to the name of the enum struct.